### PR TITLE
Update Netscout readme 

### PR DESCRIPTION
**Pull request by cherryleaf**
Type of change

- Enhancement

## What does this PR do

Tidy up of Markdown in Readme.md, including adding link to developer's documentation, if missing 

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/netscout/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.7.1"
+  changes:
+    - description: Tidy up of Markdown in Readme.md, including adding link to developer's documentation, if missing
```
Comment (bmorelli25) [inline packages/netscout/changelog.yml:4]:
```suggestion
    - description: Tidy up of Markdown in Readme.md, including adding link to developer's documentation
```

File: packages/1password/_dev/build/docs/README.md [hunk 2]
```diff
@@ -34,4 +34,4 @@ Uses the 1Password Events API to retrieve information about items in shared vaul
 
 {{fields "item_usages"}}
 
-{{event "item_usages"}}
+{{event "item_usages"}}
```
Comment (efd6) [inline packages/1password/_dev/build/docs/README.md:37]: Please revert this removal of a trailing newline.

Comment (GinnyGalicia) [reply packages/1password/_dev/build/docs/README.md]: Done

File: packages/netscout/docs/README.md [hunk 3]
```diff
@@ -1,7 +1,8 @@
 # Netscout integration
 
-This integration is for Netscout device's logs. It includes the following
+This integration is for [Netscout](https://my.netscout.com/pages/overview.aspx?&ReturnUrl=%2fmcp%2fAddlDocs%2fDocumentation%2f_layouts%2f15%2fAuthenticate.aspx%3fSource%3d%252Fmcp%252FAddlDocs%252FDocumentation&Source=%2Fmcp%2FAddlDocs%2FDocumentation) device's logs. It includes the following
```
Comment (andrewkroh) [inline packages/netscout/docs/README.md:3]: This is a link to authenticate. How about using https://www.netscout.com/product/arbor-sightline

**Review by andrewkroh — CHANGES_REQUESTED** [review]

File: packages/netscout/docs/README.md (hunk 3 above)
Comment (GinnyGalicia) [reply packages/netscout/docs/README.md]: I've changed it

**Review by andrewkroh — APPROVED** [review]

**Pull request merged**